packages/image_picker/image_picker_ios/pigeons/messages.dart

```diff
@@ -43,7 +43,7 @@ enum SourceType { camera, gallery }
 class SourceSpecification {
   SourceSpecification(this.type, this.camera);
   SourceType type;
-  SourceCamera? camera;
+  SourceCamera camera;
```

Comment:
I was going to update the code receiving this to handle the new boxed value, but realized that we would never actually send null so it was much easier to just fix the spec.